### ISSUE TITLE
Access To FreeTDS Mailing List

### ISSUE THREAD

**Issue by ChrisDStevenson** [FreeTDS/freetds#477]
Hello FreeTDS Support Team,

I am kindly requesting access to the FreeTDS Mailing List. I am trying to track down how to support Multi Subnet Failovers in a Microsoft SQL server connection using FreeTDS. If this is not yet supported, I would be interested to evaluate the options that I have, including raising a feature request.

Thank you,
Christopher Stevenson

**Comment by fziglio** [comment]
Hi Christopher, can you send me your email (even privately) ?

**Comment by ChrisDStevenson** [comment]
Hi Frediano,

Yes, it is ***@***.***

Thanks,
Christopher D. Stevenson

**Comment by fziglio** [comment]
Hi Christopher, GitHub filtered your email

**Comment by ChrisDStevenson** [comment]
Hi Frediano,
Ah let me try again: christopherd1682@gmail.com

**Comment by ChrisDStevenson** [comment]
Thank you so much! Posted to the mailing list :)

**Comment by mcepl** [comment]
Can I hijack this ticket, please?

Is email list the best support option? If yes, could you, please, subscribe mcepl AT cepl DOT eu? Thank you.

**Comment by freddy77** [comment]
done

Frediano


Il giorno dom 19 mar 2023 alle ore 08:55 Matěj Cepl <
***@***.***> ha scritto:
